test/luajit-test-init.lua

```diff
@@ -137,6 +138,13 @@ assert(type(pairs) == 'function')
 -- test case.
 loaders.override_builtin_disable()
 
+-- Increase the default Lua memory limit.
+--
+-- Some tests in the tarantool-tests suite of LuaJIT assume that
+-- for the GC64 build we have more than 4 GB of Lua memory.
+-- Increase the limit to 128 TiB (LuaJIT maximum for GC64 mode).
+alloc.setlimit(128 * 1024 * 1024 * 1024 * 1024)
```

Comment:
Nit: I would set it under `require('ffi').abi('gc64')`.

Reply:
Added:
```diff
diff --git a/test/luajit-test-init.lua b/test/luajit-test-init.lua
index 0eee1999b..b31bec43d 100644
--- a/test/luajit-test-init.lua
+++ b/test/luajit-test-init.lua
@@ -1,6 +1,7 @@
 -- Disable strict for Tarantool.
 require("strict").off()
 
+local ffi = require('ffi')
 local loaders = require('internal.loaders')
 local alloc = require('internal.alloc')
 
@@ -143,7 +144,9 @@ loaders.override_builtin_disable()
 -- Some tests in the tarantool-tests suite of LuaJIT assume that
 -- for the GC64 build we have more than 4 GB of Lua memory.
 -- Increase the limit to 128 TiB (LuaJIT maximum for GC64 mode).
-alloc.setlimit(128 * 1024 * 1024 * 1024 * 1024)
+if ffi.abi("gc64") then
+  alloc.setlimit(128 * 1024 * 1024 * 1024 * 1024)
+end
 
 -- This is workaround introduced for flaky macosx tests reported by
 -- https://github.com/tarantool/tarantool/issues/7058

```

Reply:
Thanks!